javers-spring-jpa/aop.xml

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE aspectj PUBLIC "-//AspectJ//DTD//EN" "http://www.eclipse.org/aspectj/dtd/aspectj.dtd">
```

Comment:
Can we do this without xml? I mean in Java Config

Reply:
so there is no such thing as java config at this point, its instruction for aspectj compiler (ajc) what to weave at compile time.
the link you gave has one diff with our case - we need to inject 3 beans somehow into a listener which is not created by spring, but by a EntityManager at startup. If you look at this class in build directory, you will see that in default constructor there is additional code generated by aspectJ at compile time. For that we are using spring feature - @Configured with aspectJ - https://www.baeldung.com/spring-inject-bean-into-unmanaged-objects . Same thing does spring-data-jpa with their AuditingEntityListener.
There is also another option to autowire into such objects - special class with ApplicationContext static field or having this listener with static fields and resolving them later, which in my opinion is kinda ugly. - https://stackoverflow.com/a/60651528
We could remove the xml, but then ajc will weave entire module and will find 3 classes to weave (this one, transactional decorator and javers spring data aspect), not sure if its desirable to alter their logic.